app/docs/reference/compatible-devices.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Compatible devices | Reference"
+description: "Lists devices compatible with Routegy QR codes and NFC tags"
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern devices that run either iOS or Android.
+
+
+## iOS device compatibility
+
+QR code reading capability is built into the camera app on any iPhone, iPad and iPod that runs iOS 11 or newer.
+
+NFC tag reading capability is supported in iOS 13 (and newer) on the following devices: iPhone XS, XS Max, XR, 11, 11 Pro and 11 Pro Max.
+
+## Android device compatibility
+
+QR code reading capability is built into the camera app on any device that runs Android 9 or newer. In addition to this, devices that run Android 8 can scan QR codes by using Google Lens app.
+
+NFC tag reading is built into most modern Android devices. You can check if your devices supports NFC by going into _Settings->Connected devices_ (or a similar option like _Wireless & networks_ or _Connections_). If _NFC_ is present in these settings,your device supports reading NFC tags (as long as it is turned on).
```

Comment:
1) "devices supports". Device should be singular.

2) "settings,your" - Add space after comma.